R/ns_to_percents.R

```diff
@@ -18,32 +19,32 @@
 #'   # when total_n is needed
 #'   mtcars %>%
 #'   crosstab(am, cyl) %>%
-#'   add_totals_row() %>% # add a totals row that should not be included in the denominator
+#'   add_totals("row") %>% # add a totals row that should not be included in the denominator
 #'   ns_to_percents(denom = "all", total_n = nrow(mtcars)) # specify correct denominator
   
 ns_to_percents <- function(dat, denom = "row", na.rm = TRUE, total_n = NULL){
```

Comment:
One option for a v1.0 approach: 

This is a function primarily intended to modify a tabyl, so it would be an `adorn` function, probably `adorn_percent(denom="row")` for the percent of total over rows. 

If you included both `adorn_N` and `adorn_percent` in the pipe after a `tabyl`, it would behave like `adorn_crosstab` does now, or maybe just add columns for both if that's easier. 

It sounds hard to make `tably` create an environment that could allow for multiple `adorn_` functions to be stacked. I think it's possible, but I have no idea how to do it.